src/main.cpp

```diff
@@ -301,6 +277,9 @@ int main(int argc, char **argv) {
         if (ret.stats) {
           console::EchoStats(ret.stats.value());
         }
+        if (FLAGS_verbose_query) {
+          console::EchoExecutionTimeInfo(ret.execution_time_info.value());
+        }
```

Comment:
Also, check if there is `exeuction_time_info`, some queries do not have it (e.g. `CREATE INDEX ON :label(prop);`)

Reply:
Covered